### PR TITLE
Added Inspire

### PR DESCRIPTION
**Pull request by NoahBuscher**
Inspire is a list of sites I often look to in order to get an idea of how to approach a web project. It's organized by what I use the site for, and I include a brief description with each entry.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks, but I don't accept repos that has disabled their issue tracker. Get the author to enable it again and I'll be happy to reconsider.

**Comment by NoahBuscher** [comment]
The issue tracker is now re-enabled! :+1:

File: readme.md [hunk 1]
```diff
@@ -59,6 +59,7 @@
 - [SVG](https://github.com/willianjusten/awesome-svg)
 - [KnockoutJS](https://github.com/dnbard/awesome-knockout)
 - [Dojo Toolkit](https://github.com/peterkokot/awesome-dojo)
+- [Inspire](https://github.com/NoahBuscher/Inspire)
```
Comment (sindresorhus) [inline readme.md]: I would name it `Inspiration` instead. More clear.

**Comment by NoahBuscher** [comment]
:boom: Done!

**Comment by sindresorhus** [comment]
Added :)

**Comment by NoahBuscher** [comment]
Yay! Thanks. :smile_cat:

**Pull request closed without merging**